modules/persistent-storage-csi-migration-overview.adoc

```diff
@@ -6,27 +6,6 @@
 [id="persistent-storage-csi-migration-overview_{context}"]
 = Overview
 
-With the automatic migration feature enabled, volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart CSI drivers.
+Volumes provisioned using in-tree storage plug-ins supported by this feature are migrated to their counterpart Container Storage Interface (CSI) drivers. This process does not involve any data migration, {product-title} only translates the persistent volume object in memory. As a result, the translated persistent volume object is not stored on disk, nor is its contents changed.
 
-The following drivers are supported:
-
-* Amazon Web Services (AWS) Elastic Block Storage (EBS)
-
-* OpenStack Cinder
-
-* Azure Disk
-
-* Azure File
-
-* Google Compute Engine Persistent Disk (in-tree) and Google Cloud Platform Persistent Disk (CSI)
-
-* VMware vSphere
-
-CSI automatic migration should be seamless. Enabling this feature does not change how you use all existing API objects (for example, `PersistentVolumes`, `PersistentVolumeClaims`, and `StorageClasses`).
-
-By default, automatic migration is disabled.
-
-[IMPORTANT]
-====
-CSI automatic migration will be enabled by default in a future {product-title} release, so it is highly recommended that you test it now and report any issues.
-====
+CSI automatic migration should be seamless. This feature does not change how you use all existing API objects (for example, `PersistentVolumes`, `PersistentVolumeClaims`, and `StorageClasses`).
```

Comment:
Per the ISG:

> Avoid using parentheses in general text because text in parentheses is decreased in importance.

> ! should verb
Whenever possible, rewrite in a more direct way. Do not use "should" to refer to an action that must be performed.